src/App/App.tsx

```diff
@@ -12,7 +12,7 @@ const Profile = lazy(() => import("pages/Profile"));
 const Marketplace = lazy(() => import("pages/Marketplace"));
 const Registration = lazy(() => import("pages/Registration"));
 const Donate = lazy(() => import("pages/Donate"));
-const DonateFiatThanks = lazy(() => import("pages/DonateFiatThanks"));
```

Comment:
qr crypto also now redirects to donateFiatThanks, thus rename from donateFiat- to generic donateThanks